extensions/wikia/PageShare/PageShareHooks.class.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+use Wikia\PageHeader\Button;
+
+class PageShareHooks {
+	public static function onAfterPageHeaderButtons( &$buttons ) {
+		$title = RequestContext::getMain()->getTitle();
+
+		if ( Wikia::isContentNamespace() && $title->exists() &&
```

Comment:
I think it's fine to use `exists()`. `isKnown()` would be needed if we wanted to show share button on special pages etc.